docs/remote_inference_blueprints/aleph_alpha_connector_luminous_base_embedding_blueprint.md

```diff
@@ -0,0 +1,143 @@
+# Aleph Alpha connector blueprint example for luminous-base embedding model
```

Comment:
Suggest add more details/doc links to explain this model. 
You can copy details of this PR description to this doc

Reply:
Fixed this and all subsequent comments in a new commit.